diffcore-rename.c

```diff
@@ -94,6 +106,7 @@ static struct diff_rename_src *register_rename_src(struct diff_filepair *p)
 		first = next+1;
 	}
 
+found_it:
```

Comment:
yeah, I thought about that afterwards (while not sleeping one night).  i'll rename it.